internal/experiments/experiments.go

```diff
@@ -0,0 +1,58 @@
+package experiments
+
+import (
+	"log"
+	"xk6-khorne/internal/experiments/chaos_controller"
+
+	"github.com/dop251/goja"
+	"go.k6.io/k6/js/modules"
```

Comment:
For throwing errors:
```suggestion
	"go.k6.io/k6/js/modules"
	"go.k6.io/k6/js/common"
	"errors"
```